ERROR    root:test_LastConfigStarted.py:42 ❌ Error during Last Configuration Started test: Message: no such element: Unable to locate element: {"method":"css selector","selector":"#cc-app-container-main > div.cc-app-container__main-frame.cc-grid-container > div.cc-app-container__navigation.ng-star-inserted > cc-navigation > nav > div > ul > li:nth-child(3) > ccwb-text > a"}
  (Session info: chrome=135.0.7049.114); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
#0 0x564a6b86575a <unknown>
#1 0x564a6b3184b0 <unknown>
#2 0x564a6b3699b3 <unknown>
#3 0x564a6b369bd1 <unknown>
#4 0x564a6b35d4b6 <unknown>
#5 0x564a6b38f5bd <unknown>
#6 0x564a6b35d36a <unknown>
#7 0x564a6b38f75e <unknown>
#8 0x564a6b3b55e0 <unknown>
#9 0x564a6b38f363 <unknown>
#10 0x564a6b35bd63 <unknown>
#11 0x564a6b35c9c1 <unknown>
#12 0x564a6b82aa6b <unknown>
#13 0x564a6b82e951 <unknown>
#14 0x564a6b812b62 <unknown>
#15 0x564a6b82f4c4 <unknown>
#16 0x564a6b7f713f <unknown>
#17 0x564a6b8536f8 <unknown>
#18 0x564a6b8538d6 <unknown>
#19 0x564a6b8645a6 <unknown>
#20 0x7fa86fa9caa4 <unknown>
#21 0x7fa86fb29c3c <unknown>

ERROR    root:test_LastConfigStarted.py:42 ❌ Error during Last Configuration Started test: Message: no such element: Unable to locate element: {"method":"css selector","selector":"#cc-app-container-main > div.cc-app-container__main-frame.cc-grid-container > div.cc-app-container__navigation.ng-star-inserted > cc-navigation > nav > div > ul > li:nth-child(3) > ccwb-text > a"}
  (Session info: chrome=135.0.7049.114); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
#0 0x564a6b86575a <unknown>
#1 0x564a6b3184b0 <unknown>
#2 0x564a6b3699b3 <unknown>
#3 0x564a6b369bd1 <unknown>
#4 0x564a6b35d4b6 <unknown>
#5 0x564a6b38f5bd <unknown>
#6 0x564a6b35d36a <unknown>
#7 0x564a6b38f75e <unknown>
#8 0x564a6b3b55e0 <unknown>
#9 0x564a6b38f363 <unknown>
#10 0x564a6b35bd63 <unknown>
#11 0x564a6b35c9c1 <unknown>
#12 0x564a6b82aa6b <unknown>
#13 0x564a6b82e951 <unknown>
#14 0x564a6b812b62 <unknown>
#15 0x564a6b82f4c4 <unknown>
#16 0x564a6b7f713f <unknown>
#17 0x564a6b8536f8 <unknown>
#18 0x564a6b8538d6 <unknown>
#19 0x564a6b8645a6 <unknown>
#20 0x7fa86fa9caa4 <unknown>
#21 0x7fa86fb29c3c <unknown>

ERROR    root:test_LastConfigStarted.py:42 ❌ Error during Last Configuration Started test: Message: no such element: Unable to locate element: {"method":"css selector","selector":"#cc-app-container-main > div.cc-app-container__main-frame.cc-grid-container > div.cc-app-container__navigation.ng-star-inserted > cc-navigation > nav > div > ul > li:nth-child(3) > ccwb-text > a"}
  (Session info: chrome=135.0.7049.114); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
#0 0x564a6b86575a <unknown>
#1 0x564a6b3184b0 <unknown>
#2 0x564a6b3699b3 <unknown>
#3 0x564a6b369bd1 <unknown>
#4 0x564a6b35d4b6 <unknown>
#5 0x564a6b38f5bd <unknown>
#6 0x564a6b35d36a <unknown>
#7 0x564a6b38f75e <unknown>
#8 0x564a6b3b55e0 <unknown>
#9 0x564a6b38f363 <unknown>
#10 0x564a6b35bd63 <unknown>
#11 0x564a6b35c9c1 <unknown>
#12 0x564a6b82aa6b <unknown>
#13 0x564a6b82e951 <unknown>
#14 0x564a6b812b62 <unknown>
#15 0x564a6b82f4c4 <unknown>
#16 0x564a6b7f713f <unknown>
#17 0x564a6b8536f8 <unknown>
#18 0x564a6b8538d6 <unknown>
#19 0x564a6b8645a6 <unknown>
#20 0x7fa86fa9caa4 <unknown>
#21 0x7fa86fb29c3c <unknown>

ERROR    root:test_LastConfigStarted.py:42 ❌ Error during Last Configuration Started test: Message: no such element: Unable to locate element: {"method":"css selector","selector":"#cc-app-container-main > div.cc-app-container__main-frame.cc-grid-container > div.cc-app-container__navigation.ng-star-inserted > cc-navigation > nav > div > ul > li:nth-child(3) > ccwb-text > a"}
  (Session info: chrome=135.0.7049.114); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
#0 0x564a6b86575a <unknown>
#1 0x564a6b3184b0 <unknown>
#2 0x564a6b3699b3 <unknown>
#3 0x564a6b369bd1 <unknown>
#4 0x564a6b35d4b6 <unknown>
#5 0x564a6b38f5bd <unknown>
#6 0x564a6b35d36a <unknown>
#7 0x564a6b38f75e <unknown>
#8 0x564a6b3b55e0 <unknown>
#9 0x564a6b38f363 <unknown>
#10 0x564a6b35bd63 <unknown>
#11 0x564a6b35c9c1 <unknown>
#12 0x564a6b82aa6b <unknown>
#13 0x564a6b82e951 <unknown>
#14 0x564a6b812b62 <unknown>
#15 0x564a6b82f4c4 <unknown>
#16 0x564a6b7f713f <unknown>
#17 0x564a6b8536f8 <unknown>
#18 0x564a6b8538d6 <unknown>
#19 0x564a6b8645a6 <unknown>
#20 0x7fa86fa9caa4 <unknown>
#21 0x7fa86fb29c3c <unknown>

ERROR    root:test_LastConfigStarted.py:42 ❌ Error during Last Configuration Started test: Message: no such element: Unable to locate element: {"method":"css selector","selector":"#cc-app-container-main > div.cc-app-container__main-frame.cc-grid-container > div.cc-app-container__navigation.ng-star-inserted > cc-navigation > nav > div > ul > li:nth-child(3) > ccwb-text > a"}
  (Session info: chrome=135.0.7049.114); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
#0 0x564a6b86575a <unknown>
#1 0x564a6b3184b0 <unknown>
#2 0x564a6b3699b3 <unknown>
#3 0x564a6b369bd1 <unknown>
#4 0x564a6b35d4b6 <unknown>
#5 0x564a6b38f5bd <unknown>
#6 0x564a6b35d36a <unknown>
#7 0x564a6b38f75e <unknown>
#8 0x564a6b3b55e0 <unknown>
#9 0x564a6b38f363 <unknown>
#10 0x564a6b35bd63 <unknown>
#11 0x564a6b35c9c1 <unknown>
#12 0x564a6b82aa6b <unknown>
#13 0x564a6b82e951 <unknown>
#14 0x564a6b812b62 <unknown>
#15 0x564a6b82f4c4 <unknown>
#16 0x564a6b7f713f <unknown>
#17 0x564a6b8536f8 <unknown>
#18 0x564a6b8538d6 <unknown>
#19 0x564a6b8645a6 <unknown>
#20 0x7fa86fa9caa4 <unknown>
#21 0x7fa86fb29c3c <unknown>

ERROR    root:test_LastConfigStarted.py:47 ❌ Last Configuration Started Test failed after 5 attempts.
ERROR    root:VerifyPersonalizationAndCapture.py:87 ❌ Image not found in the specified selector. Error: Message: 

ERROR    root:VerifyPersonalizationAndCapture.py:18 ❌ Error attaching screenshot to Allure: [Errno 2] No such file or directory: '/home/runner/work/QA-Perso/QA-Perso/Tests/Last Configuration Started_attempt_2.png'
ERROR    root:QAAppAllure.py:172 ❌ Test 'Last Configuration Started' failed.